base/applications/mspaint/lang/tr-TR.rc

```diff
@@ -268,9 +269,9 @@ BEGIN
     IDS_ITALIC "İtalik"
     IDS_UNDERLINE "Altı Çizgili"
     IDS_VERTICAL "Düşey"
-    IDS_PRINTRES "%d x %d pixel/cm"
-    IDS_CANTPASTE "Failed to paste from the clipboard. The data format is either incorrect or not supported."
-    IDS_SAVEERROR "Failed to save the bitmap to file:\n\n%s"
-    IDS_CANTSENDMAIL "Failed to send a mail."
-    IDS_LOSECOLOR "The color information will be lost in this operation. Are you sure to proceed?"
+    IDS_PRINTRES "%d x %d piksel/cm"
+    IDS_CANTPASTE "Panodan yapıştırılamadı. Veri formatı bozuk ya da desteklenmiyor."
+    IDS_SAVEERROR "Bitmap'i dosyaya:\n\n%s kaydederken bir hata oluştu."
```

Comment:
Alternative text:

“Bitmap’i dosyaya kaydederken bir hata oluştu. Dosya: \n\n%s”

Reply:
This is a good proposal imho, but please **without** the space before the `\n\n`.